package.json

```diff
@@ -7,8 +7,14 @@
     "type": "git",
     "url": "git://github.com/fin-hypergrid/core.git"
   },
-  "author": "SWirts, JEiten, DJones, NMichaud",
-  "license": "MIT",
+  "author": "Steve Wirts (https://github.com/stevewirts)",
+  "contributors": [
+    "Jonathan Eiten (https://github.com/joneit)",
+    "Dwayne Jones (https://github.com/dwaynekj)",
+    "Naveen Michaud-Agrawal (https://github.com/nmichaud)",
+    "Hugo Nogueira (https://github.com/hugoeanogueira)"
+  ],
+  "license": "MIT (as modified in LICENSE file)",
```

Comment:
What does "as modified in License File" mean? MIT License is what it is, not clear how it can be modified?